letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -319,6 +319,32 @@ else
   BOOTSTRAP_VERSION="Unknown 1"
 fi
 
+PrevBootstrapVersion() {
+  if [ -f $VENV_BOOTSTRAP_VERSION ]; then
+    cat $VENV_BOOTSTRAP_VERSION
+  elif grep -Fqx "$BOOTSTRAP_VERSION" << "UNLIKELY_EOF"
+NO_BOOTSTRAP 1
+BootstrapDebCommon 1
```

Comment:
So the thought here was that we want our current bootstrap scripts to be treated specially. If there's no bootstrap version saved to disk but the bootstrap script we're considering running is from before we landed this PR, we probably don't need to rebootstrap with the thinking that the venv was from before we made this change. New bootstrap scripts we make in the future though should always have their version saved.

I very indirectly touched on this [here](https://github.com/certbot/certbot/pull/4971/files#diff-a4881cc6ba1c9dcaeb3962d88a0c369dR763), but I'm happy to expand that comment and/or move it to before this list.